docs/using/security-practices.md

```diff
@@ -306,43 +306,4 @@ makes client-side confinement largely moot for the moment.
 
 ## Case Studies
 
-We've already seen Sandstorm's security model mitigate bugs in apps. Here are some examples.
-
-### Etherpad
-
-Etherpad has, over the last several months, disclosed several vulnerabilities falling into a few categories:
-
-* Path injection attacks allowing anyone to download arbitrary files
-  from the server's filesystem (such as Etherpad's own database,
-  containing credentials).
-
-* Information leaks allowing someone with access to one document on
-  the server to discover and read other documents.
-
-* Various other minor information leaks of user and system
-  information.
-
-None of these problems have affected Etherpad on Sandstorm, because
-Sandstorm runs every Etherpad document in a separate container, and
-because Sandstorm implements authentication, sharing, and access
-control.
-
-### Wordpress
-
-The Sandstorm port of Wordpress separates the Wordpress admin
-interface from the publicly-facing published web site. The admin
-interface is accessed through Sandstorm, whereas the published site is
-served as read-only content. The Wordpress may not even be running
-when public content is served; Sandstorm simply returns the content
-from disk.
-
-As a result, many recent Wordpress security vulnerabilities are
-mitigated by Sandstorm: since regular visitors to the site have no
-ability to interact with Wordpress software, they have no ability to
-exploit it. Several recent Wordpress security vulnerabilities are
-therefore irrelevant to Sandstorm users.
-
-(Admittedly, this protection currently comes at the cost of disabling
-features like public comments. We plan to address this in the
-future. For many use cases, though, this restriction is perfectly
-acceptable.)
+See [Security non-events](security-non-events) for examples of security problems in apps and in the Linux kernel which were mitigated by Sandstorm.
```

Comment:
+1, good link, nice to see the security-practices.md and the non-issues/non-events page get merged.